packages/components/src/FormatDate/FormatDate.mdx

```diff
@@ -1,13 +1,14 @@
----
-name: FormatDate
-menu: Components
-route: /components/date-formatter
+## import {Text} from "../Text";
+
+name: FormatDate menu: Components route: /components/date-formatter
+
```

Comment:
Something happened here with the `frontmatter`

![image](https://user-images.githubusercontent.com/46460840/107400014-a0db7700-6abe-11eb-8a8e-2d6469b261f3.png)